docs/workflow-controller-configmap.yaml

```diff
@@ -225,8 +225,8 @@ data:
     metricsTTL: "10m"
     # IgnoreErrors is a flag that instructs prometheus to ignore metric emission errors. Default is "false"
     ignoreErrors: false
-    # Use a self-signed cert for TLS, default false
-    secure: false
+    # Use a self-signed cert for TLS, default true
+    secure: true
 
     # DEPRECATED: Legacy metrics are now removed, this field is ignored
     disableLegacy: false
```

Comment:
probably makes sense to remove this one of your PRs as well

Reply:
This could be separated from this stack.

We'd break compatibility if we remove it from the golang struct too, but I'm inclined to do that anyway. Thoughts?

Reply:
> This could be separated from this stack.

Yea can be, if it breaks compat then it'd be more of a timing thing to release together.

> We'd break compatibility if we remove it from the golang struct too, but I'm inclined to do that anyway. Thoughts?

It is? I thought it was just dated docs at this point 😅
But yes still worth removing, though I guess we might need an upgrade note?